__tests__/app.test.js

```diff
@@ -0,0 +1,40 @@
+const app = require("../app");
+const request = require("supertest");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+
+const data = require("../db/data/test-data/");
+
+beforeEach(() => seed(data));
+
+afterAll(() => db.end());
+
+describe("Test /api/topics", () => {
+  test("GET 200: Should return an array of topics to the client", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.length).toBe(3);
+      });
+  });
+  test("GET 200: Each item in the array of topics should have a string 'Description' and a string 'slug'", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((response) => {
+        expect(typeof response.body[0].description).toBe("string");
+        expect(typeof response.body[0].slug).toBe("string");
+        expect(typeof response.body[1].description).toBe("string");
+        expect(typeof response.body[1].slug).toBe("string");
+        expect(typeof response.body[2].description).toBe("string");
+        expect(typeof response.body[2].slug).toBe("string");
+      });
+  });
+
+  test("ERROR: 404 , the url is misstyped", () => {
```

Comment:
You don't have a custom 404 so I think this test will have passed straight away? 😄 This is because Express handles this. No test needed unless making your own handler.